Gcpe.Hub.API.Tests/ControllerTests/SocialMediaControllerTests.cs

```diff
@@ -0,0 +1,292 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoMapper;
+using FluentAssertions;
+using Gcpe.Hub.API.Controllers;
+using Gcpe.Hub.API.Data;
+using Gcpe.Hub.API.ViewModels;
+using Gcpe.Hub.Data.Entity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Gcpe.Hub.API.Tests.ControllerTests
+{
+    public class SocialMediaControllerTests
+    {
+        private Mock<ILogger<SocialMediaController>> logger;
+        private HubDbContext context;

```

Comment:
I don't think you need the context anymore

Reply:
I'm using it throughout the tests to add posts to the db